### DISCUSSION TITLE
Separation of to-do items

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#6679] [Discussion]
When working on a collaborative project, it’s important to be clear about what’s happening next and what’s just a suggestion. This is why we have to-do lists. However, we use a setup that more or less gives us limited options for planning things.

### Milestones for releases

What we currently do is add the things we want to plan for a release to a milestone, which is a good idea, but what if we want to consider something and work on it at some undefined time in the future? Where does it go?

That’s why I suggest one of the following:

### A “to do” label

Labeling tasks is a simple and effective way to mark something as “planned”, without necessarily specifying a due date.

### Priority means we’re doing this

Adding a priority label could also function as a “to do” label.

### A Project

A bigger change and potential step up would be to move to a Project acting as a board for what’s happening next. The two previous ideas could be combined as [iteration fields](https://docs.github.com/en/issues/planning-and-tracking-with-projects/understanding-fields/about-iteration-fields) would act as our milestones and **having an item in the project at all would act as the “to do” label**. This would be a _replacement_ for the milestones we’ve been using since the start. For once we might be successful in using GitHub Projects.

However, the other options might be simpler and seem to do most of what we need, but there are some benefits to using Projects such as grouping and sorting, [creating draft issues](https://docs.github.com/en/issues/planning-and-tracking-with-projects/managing-items-in-your-project/adding-items-to-your-project#creating-draft-issues) and adding issues from across repositories, and custom fields that we can create (importance, estimate, etc.). What do you think? Also note that projects have a 1,200-item limit, but we probably won’t get too close to that. In fact, we’ll probably try to stay below a threshold even lower.

## Related:

- #6618
- #5909

Any thoughts, @ScratchAddons/admins? Has this been a problem? What would you do if it were?

**Comment by WorldLanguages** [comment]
> what if we want to consider something and work on it at some undefined time in the future? Where does it go?

I've sometimes used the prio labels and/or the backlog milestone for those things.
